fern/pages/sdks/introduction/what-is-an-sdk.mdx

```diff
@@ -0,0 +1,255 @@
+            `v1/text-to-speech/${encodeURIComponent(voiceId)}`
+        ),
+        method: "POST",
+        headers: {
+            "xi-api-key":
+                (await core.Supplier.get(this._options.apiKey)) != null
+                    ? await core.Supplier.get(this._options.apiKey)
+                    : undefined,
+            "X-Fern-Language": "JavaScript",
+            "X-Fern-SDK-Name": "elevenlabs",
+            "X-Fern-SDK-Version": "v0.6.0",
+            "X-Fern-Runtime": core.RUNTIME.type,
+            "X-Fern-Runtime-Version": core.RUNTIME.version,
+        },
+        contentType: "application/json",
+        queryParameters: _queryParams,
+        body: _body,
+        responseType: "streaming",
+        timeoutMs: requestOptions?.timeoutInSeconds != null ? requestOptions.timeoutInSeconds * 1000 : 60000,
+        maxRetries: requestOptions?.maxRetries,
+    });
+    if (_response.ok) {
+        return _response.body;
+    }
+
+    if (_response.error.reason === "status-code") {
+        switch (_response.error.statusCode) {
+            case 422:
+                throw new ElevenLabs.UnprocessableEntityError(
+                    _response.error.body as ElevenLabs.HttpValidationError
+                );
+            default:
+                throw new errors.ElevenLabsError({
+                    statusCode: _response.error.statusCode,
+                    body: _response.error.body,
+                });
+        }
+    }
+
+    /** ...more error handling... */
+
+}
+```
+</CodeBlock>
+<CodeBlock title="ElevenLabs.TextToSpeechRequest">
+```TypeScript
+export interface TextToSpeechRequest {
+    /** You can turn on latency optimizations at some cost of quality. The best possible final latency varies by model. */
+    optimize_streaming_latency?: ElevenLabs.OptimizeStreamingLatency;
+    /** The output format of the generated audio. */
+    output_format?: ElevenLabs.OutputFormat;
+    /** The text that will get converted into speech. */
+    text: string;
+    /** Identifier of the model that will be used, you can query them using GET /v1/models. The model needs to have support for text to speech, you can check this using the can_do_text_to_speech property. */
+    model_id?: string;
+    /** Voice settings overriding stored setttings for the given voice. They are applied only on the given request. */
+    voice_settings?: ElevenLabs.VoiceSettings;
+    /** A list of pronunciation dictionary locators (id, version_id) to be applied to the text. They will be applied in order. You may have up to 3 locators per request */
+    pronunciation_dictionary_locators?: ElevenLabs.PronunciationDictionaryVersionLocator[];
+}
+```
+</CodeBlock>
+
+</CodeBlocks>
+
+### One API, many SDKs
+
+SDKs are platform-specific, so each language requires its own. Here's the same ElevenLabs API endpoint called via SDKs in multiple languages:
+
+<CodeBlocks maxLines={10}>

```

Comment:
Can you add TypeScript and C#? If you need to generate the SDK locally to see what the code looks like in each language, you should do that. Doesn't look like Ruby, Java, or Go are using SDK code (instead they're using HTTP calls directly now)

Order should be: TypeScript, Python, Go, Java, Ruby, C#